### PR TITLE
Move Generic Language Eggs

### PR DESCRIPTION
**Pull request by parkervcp**
This moves all the generic language eggs into a separate folder

No changes to the eggs except for their names and removing anything pertaining to bots. This excludes the python egg where a bot repo was used for testing.

### All Submissions:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:
1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [ ] Have you tested your Egg changes?

### CODE REVIEW

**Comment by Software-Noob** [comment]
Nodemon egg is a generic too for nodejs apps, move it as well?

The readme of java, node, and go eggs have their old discord-{lang} name titles. That's probably something that should be changed as well to prevent confusion.

The main readme file must also be modified to remove the old discord folder path and display a new generic one. The discord folder readme must be modified to remove the non-existing discord eggs.

The generic folder by itself is not very descriptive to end users. Maybe `generic-code`?

Otherwise, LGTM

File: generic/golang/egg-golang-generic.json [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-    "name": "discordgo generic",
+    "exported_at": "2022-07-25T14:20:37-04:00",
+    "name": "golang generic",
     "author": "parker@parkervcp.com",
-    "description": "A generic golang egg.\r\n\r\nMeant to be customized before it is used.",
-    "image": "quay.io\/parkervcp\/pterodactyl-images:base_debian",
+    "description": "A generic golang egg that compiles the application and saves an executable\r\n\r\nMeant to be customized before it is used.",
+    "features": null,
+    "docker_images": {
+        "quay.io\/parkervcp\/pterodactyl-images:base_debian": "quay.io\/parkervcp\/pterodactyl-images:base_debian"
```
Comment (gOOvER) [inline generic/golang/egg-golang-generic.json:13]: This should be a ghcr image

Comment (gOOvER) [inline generic/golang/egg-golang-generic.json:13]: This should be a ghcr image

**Review by gOOvER — CHANGES_REQUESTED** [review]

**Review by QuintenQVD0 — APPROVED** [review]
Looks good to me!

**Pull request merged**